src/App.jsx

```diff
@@ -1,36 +1,36 @@
 import React, { Component } from 'react';
-import './App.css';
+import './App.scss';
```

Comment:
The way create-react-app set things up, it automatically compiles (and watches and re-compiles) Sass files into CSS files, so this should remain an import of the compiled CSS.